packages/salesforcedx-apex-debugger/src/core/sessionService.ts

```diff
@@ -75,19 +75,26 @@ export class SessionService {
         .build(),
       { cwd: this.project }
     ).execute();
-    const result = await this.getIdFromCommandResult(execution);
```

Comment:
getIdFromCommandResult is private and isn't used anymore, so you'll have some tslint errors. Could you run the tslint command on apex-debugger package?